src/js/form-render.js

```diff
@@ -82,10 +82,34 @@ class FormRender {
           className: 'rendered-form',
         })
         this.appendChild(renderedFormWrap)
+  
         fields.forEach(field => {
-          renderedFormWrap.appendChild(field)
-          field.dispatchEvent(events.fieldRendered)
+          // Determine if rows are being used. If so, create the row and append to its row-{group}
+          // If the fields have row-, create & append to the appropriate row
+          const classes = field.className.split('row-');
```

Comment:
Putting a row-{number} was only for example. You might want to name the row to something more meaningful like row-employeeInfo. Then again, maybe the row- identifier really doesn't matter at all.  It really depends how it is implemented. For example if the row & columns could be configured via a drag & drop user interface then perhaps the row- identifier could be auto-generated. 

For now it would be best to have the regex match against row-{word} up until a space. So it might look like row-([^\s]+)